charts/k8s-monitoring/docs/Customizations.md

```diff
@@ -0,0 +1,107 @@
+# Customizing the Configuration
+
+While this chart provides a capable configuration for Kubernetes Cluster infrastructure monitoring, you can use this document to customize your configuration and understand how these customizations affect it.
+
+## Overview
+
+Each telemetry data source (e.g. metrics, logs, events, traces) typically goes through the same process:
+
+1. Discovery - How does the collector find the data source?
+2. Scraping - How should the data get gathered?
+3. Processing - What work needs to be done to the data?
+4. Delivery - Where should the data be sent for storage?
+
+## Metrics
+
+Metrics are typically gathered using Prometheus-style scraping.
+
+### Discovery customizations
+
+These fields affect how metric targets are discovered. Typically, you'll make changes to these values if the Grafana Agent is not detecting the target.
```

Comment:
```suggestion
These fields affect how metric targets are discovered. If the Grafana Agent is not detecting the target, make changes to these values.
```

Reply:
In this section, do we need to give some examples of what issues users are trying to solve, such as reducing metrics? For eample, why would I use relabeling for one source as opposed to all?